converter/dicom-cast/docs/workingWithPrivateLink.md

```diff
@@ -0,0 +1,24 @@
+# Configuration steps for dicomast to work with privatelink enabled dicom service
```

Comment:
nit i think private link should be two words - https://azure.microsoft.com/en-us/products/private-link